lib/registries/addon/components/registries-states/is-public/template.hbs

```diff
@@ -0,0 +1,33 @@
+<div local-class="WithdrawDropdown">
+    <p local-class='header'>{{t 'overview.withdraw.withdraw'}}</p>
+    <p>{{t 'overview.withdraw.warning'}}</p>
+    <p><strong>{{t 'overview.withdraw.withdrawal_justification_label'}}</strong></p>
+    <div class="form-group">
+        {{textarea
+            class="form-control"
+            rows="4"
+            required=true
+            value=this.withdrawalJustification
+        }}
+    </div>
+    <p>{{t 'overview.withdraw.random_scientist_x' x=this.scientistName}}</p>
+    <div class="row">
```

Comment:
Prefer `local-class` and `display: flex` over bootstrap's styling. I use [this website](https://yoksel.github.io/flex-cheatsheet/) as a reference.

For this case replacing `row` with the following and removing the `col-sm-6` classes should work 
```css
{
    display: flex;
     justify-content: space-between;
    width: 100%;
}
```